torch/_dynamo/utils.py

```diff
@@ -3106,3 +3107,39 @@ def get_user_object_from_id(obj_id):
 def store_user_object_weakref(obj):
     obj_id = id(obj)
     user_obj_id_to_weakref[obj_id] = weakref.ref(obj)
+
+
+class CompileTimeInstructionCounter:
```

Comment:
Why is this "CompileTimeInstructionCounter" and not just "CpuInstructionCounter"? It has no actual relationship to compile time, does it? (Other than we intend to use it for such)

Reply:
This should not be used around anything else but counting compile time instructions since its accessed globally 
as CompileTimeInstructionCounter.value()

Reply:
if we call it CpuInstructionCounter we can create an a global instance and 
compileTimeInstructionCounter = CpuInstructionCounter()
but i dont see right now a reason for generalizing it